api/api/controllers/agents_controller.py

```diff
@@ -54,7 +53,9 @@ def delete_agents(pretty=False, wait_for_complete=False, ids=None, purge=None, s
     return data, 200
 
 
-def get_all_agents(pretty=False, wait_for_complete=False, offset=0, limit=None, select=None, sort=None, search=None, status=None, q='', older_than=None, os_platform=None, os_version=None, os_name=None, manager=None, version=None, group=None, node_name=None, name=None, ip=None):  # noqa: E501
+def get_all_agents(pretty=False, wait_for_complete=False, offset=0, limit=None, select=None, sort=None, search=None,
```

Comment:
`limit=None` will return all elements available in the API. I think it's better to set its default value using this variable:
https://github.com/wazuh/wazuh/blob/8a34d85662154d324cd4c01455e205dc34367f04/framework/wazuh/common.py#L114
```suggestion
def get_all_agents(pretty=False, wait_for_complete=False, offset=0, limit=common.database_limit, select=None, sort=None, search=None,
```

Reply:
That default value is already set in spec.yaml, so I think it's better to preserve that value in one place only to avoid future ambiguity. Thoughts?

Reply:
You're right, if all default values will be defined in the API spec, then we can dismiss this.